doc/branch.rst

```diff
@@ -41,33 +40,30 @@ To measure branch coverage, run coverage.py with the ``--branch`` flag::
 
 When you report on the results with ``coverage report`` or ``coverage html``,
 the percentage of branch possibilities taken will be included in the percentage
-covered total for each file.  The coverage percentage for a file is the actual
-executions divided by the execution opportunities.  Each line in the file is an
-execution opportunity, as is each branch destination.
+covered total for each file. Each line in the file is an execution opportunity,
+as is each branch destination.
 
 The HTML report gives information about which lines had missing branches. Lines
-that were missing some branches are shown in yellow, with an annotation at the
-far right showing branch destination line numbers that were not exercised.
```

Comment:
I'm not sure why you removed "at the far right".  I thought people might not notice it at the edge that usually has no information, so I wanted to direct their attention there.